.github/workflows/publish-pacts.yaml

```diff
@@ -0,0 +1,50 @@
+name: Publish Pacts
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ dev ]
+
+jobs:
+  setup-and-test:
+    runs-on: ubuntu-latest
+    outputs:
+      nb-b64: ${{ steps.encode.outputs.nb-b64 }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
+
+      - name: Install yarn
+        run: |
+          yarn install
+
+      - name: Run tests
+        run: yarn test
```

Comment:
In this workflow, we only _need_ to run the Pact/contract tests, correct? Is there a way we could distinguish those from other tests and not run more tests than we need to here?